snuba/datasets/schema.py

```diff
@@ -196,7 +196,7 @@ def __get_local_destination_table_name(self) -> str:
 
     def __get_table_definition(self, name: str, source_table_name: str, destination_table_name: str) -> str:
         return """
-        CREATE MATERIALIZED VIEW %(name)s TO %(destination_table_name)s (%(columns)s) AS %(query)s""" % {
+        CREATE MATERIALIZED VIEW IF NOT EXISTS %(name)s TO %(destination_table_name)s (%(columns)s) AS %(query)s""" % {
```

Comment:
Did you try to run this to ensure Clickhouse lieks this syntax ?

Reply:
Yas